modules/rosa-using-sts.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-understanding.adoc
+
+[id="rosa-using-sts_{context}"]
+= Using the AWS Security Token Service
+
+The Amazon Web Services (AWS) Security Token Service (STS) is a global web service that provides short-term credentials for IAM or federated users. You can use AWS STS with {product-title} (ROSA) to allocate temporary, limited-privilege credentials for component-specific IAM roles. The service enables cluster components to make AWS API calls using secure cloud resource management practices.
+
+You can use the `rosa` CLI to create the IAM role, policy and identity provider resources that are required for ROSA clusters that use STS. For more information, see the _About IAM resources for clusters that use STS_ section.
```

Comment:
Is `About IAM resources for clusters that use STS` supposed to be a url? There is no url, it is only italic.

Reply:
@xueli181114 thank you for your review and for this comment.

At the moment we cannot include cross-references in modules, so this is just an italicised reference to the "About IAM resources for clusters that use STS" section. I considered adding an additional resources section at the base of the assembly to include the cross-reference, but it didn't seem appropriate to highlight that one page only, given that the assembly details a broad set of ROSA capabilities.

Reply:
But `About IAM resources for clusters that use STS` section is not even in the same page, how can users find it?

Reply:
@xueli181114 after a review within the SD docs team, I have removed the reference line for now and we are going to consider this as part of a broader review of this assembly and the welcome page. This will be done outside of the context of this PR.